contracts/Events/EventManager.sol

```diff
@@ -1,5 +1,4 @@
-pragma solidity ^0.4.24;
-import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
+pragma solidity ^0.4.4;
```

Comment:
Change this back to ^0.4.24 please. We try to indicate which version of Solidity we compiled these contracts with via the pragma.